CONTRIBUTING.md

```diff
@@ -78,6 +78,8 @@ A [`Makefile`](./Makefile) is set up to run tests.
 
 ```sh make test ```
```

Comment:
We should make this a multiline fenced block and remove the odd space at the end.

Just a minor nitpick

Reply:
Oops, realized that was there already, must have been line squashed during some formatting. If you want to, feel free to fix them up and give them some breathing room

Reply:
Sure, I can clean that up. Not a problem 😄.

Reply:
Done.